target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -212,3 +213,4 @@ esac
 board_config_flush
 
 exit 0
+
```

Comment:
```suggestion
```
Please do not add this unrelated change.

Reply:
Thank you